submissions/goodnessOlakanmi/html-forms/Product-page.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Omman Calzature</title>
+	</head>
+<body>
+	<nav role="navigation">
+		<a href="index.html">OMMAN CALZATURE</a>
+		<ul class="nav">
+			<li><a href="#">Home</a></li>
+			<li><a href="#">About Us</a></li>
+			<li><a href="#">News</a></li>
+			<li><a href="#">Gallery</a></li>
+			<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Contact-Form-Page.html">Contact Form</a></li>
+			<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Product-page.html">Product Page</a></li>
```

Comment:
The link is too long. You can simply do 'product.page.html' instead of starting from C: . Since they are both in the same folder, it will redirect as expected. Also make the same change for the contact form page.

Reply:
I have edited it